include/psp2kern/avcodec/jpegenc.h

```diff
@@ -6,7 +6,7 @@
 #ifndef _PSP2_AVCODEC_JPEGENC_H_
 #define _PSP2_AVCODEC_JPEGENC_H_
```

Comment:
additionally, can you change this definition check to

```suggestion
#ifndef _PSP2_KERNEL_AVCODEC_JPEGENC_H_
#define _PSP2_KERNEL_AVCODEC_JPEGENC_H_
```

Reply:
Oh, i think KERNEL prefix collides with psp2/kernel headers definition check prefix. Hm, should i make another PR or fix it here?

Reply:
There are around ten of other cases like this too.

Reply:
sure. you can make another pr :)